go.mod

```diff
@@ -27,8 +29,11 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/zalando/go-keyring v0.0.0-20200121091418-667557018717
 	go.uber.org/zap v1.19.0
+	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
 	golang.org/x/net v0.0.0-20210428140749-89ef3d95e781
+	golang.org/x/sys v0.0.0-20210616094352-59db8d763f22 // indirect
 	golang.org/x/text v0.3.6
+	golang.org/x/tools v0.1.5 // indirect
```

Comment:
Do we really need these go modules? I didn't find any import statements.

Reply:
![toolsindirect](https://user-images.githubusercontent.com/53268258/131523394-f2165e1e-4733-4d05-9ffe-38091b1492da.PNG)
I had run 'go mod tidy' and the go.mod did include golang.org/x/tools. And it was required by another dependency.